scimma_admin/scimma_admin/settings.py

```diff
@@ -42,6 +42,17 @@ def get_localdev_secret(name):
     print(cp.sections())
     return cp["secrets"][name]
 
+# ELB is extremely picky about the headers on HTTP 301 responses for them to be correctly passed
+# back to the client. This custom middleware tries to keep it happy.
```

Comment:
Is there any documentation or something you can link to to describe this behavior? The response headers make very little sense to me: "I'm UTF-8 encoding the following zero bytes, and they have any MIME type at all" is a very strange thing to need to tell ELB.

Reply:
There is documentation at https://aws.amazon.com/premiumsupport/knowledge-center/load-balancer-http-502-errors/ , https://stackoverflow.com/a/17988855 was more directly helpful. If I remember correctly, I tested adding only the explicit `Content-Length` header, and that still wasn't enough to satisfy ELB, even though changing the `Content-Type` header seems extremely bizzare to me as well.